encrypted-media-respec.html

```diff
@@ -1338,7 +1338,7 @@
           <p>The following step is run:</p>
           <ol>
             <li>
-              <p>For each <a>MediaKeySession</a> created by the <var>media keys</var> on which the <a def-id="session-close-algorithm"></a> algorithm has not been run, run the <a def-id="session-close-algorithm"></a> algorithm.</p>
+              <p>For each <a>MediaKeySession</a> created by the <var>media keys</var> on which the <a def-id="session-closed-algorithm"></a> algorithm has not been run, run the <a def-id="close-session-algorithm"></a> algorithm.</p>
```

Comment:
The Close All Sessions algorithm appears to _only_ be run when the CDM is unavailable. Two things:

1. We should rename this to "CDM Unavailable algorithm" or something like that to make this clear.
1. It doesn't make sense to call Close Session, which uses the CDM to destroy keys if the CDM is unavailable. It seems we should call Session Closed instead.
 * We might want to be clear that the MediaKey
algorithm" or something like that to make this clear.
1. It doesn't make sense to call Close Session, which uses the CDM to destroy keys if the CDM is unavailable. It seems we should call Session Closed instead.
 * We might want to be clear that the MediaKeys is unusable and thus any keys (i.e. in memory) are inaccessible to the app.